commands/daemon.go

```diff
@@ -122,8 +123,23 @@ func getRepo(req *cmds.Request) (repo.Repo, error) {
 	return repo.OpenFSRepo(getRepoDir(req))
 }
 
-func runAPIAndWait(ctx context.Context, node *node.Node, config *config.Config, req *cmds.Request) error {
-	api := impl.New(node)
+func runAPIAndWait(ctx context.Context, nd *node.Node, config *config.Config, req *cmds.Request) error {
+	api := impl.New(nd)
+	blockTime, mineDelay := nd.MiningTimes()
+	protocolAPI := bapi.New(
+		nd.AddNewBlock,
+		nd.Blockstore,
+		nd.CborStore(), nd.OnlineStore,
```

Comment:
for some reason this one and minedely are two to a line

Reply:
it was just to follow the declaration format. I can put in the newlines.